templates/releases/releases.html

```diff
@@ -28,7 +28,7 @@
                     {%- endif -%}
 
                     <li>
-                        <a href="{{ link }}" class="release">
+                        <a href="{{ link | safe }}" class="release">
```

Comment:
Need to escape `name` and `version` in link. Although maybe since crates.io disallows `/` in names we can just assume all of these don't need to be escaped? I just think we should be consistent.